src/main/java/athleticli/ui/CommandName.java

```diff
@@ -15,12 +15,16 @@ public class CommandName {
     public static final String COMMAND_ACTIVITY = "activity";
     public static final String COMMAND_CYCLE = "cycle";
     public static final String COMMAND_SWIM = "swim";
```

Comment:
One suggestion is to change activity keywords to `add-activity`, `add-cycle`, etc to standardize the commands across the board.